web-animations/interfaces/AnimationTimeline/document-timeline.html

```diff
@@ -20,6 +21,17 @@
 }, 'document.timeline identity tests');
 
 promise_test(function(t) {
+  var baseTime = document.timeline.currentTime;
+  var iframeBaseTime = iframe.contentDocument.timeline.currentTime;
```

Comment:
Added var iframe.

It would be possible to create the iframe dynamically but it may introduce weird timing issues, I'm inclined to leave it the way it is as there's not much gain from being dynamic.